apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -104,6 +105,43 @@ public async Task Overview()
             }
         }
 
+        [Fact]
+        public async Task CustomCallInvoker()
+        {
+            ChannelCredentials credentials = ChannelCredentials.Insecure;
+
+            // Sample: CustomCallInvoker
+            // Create a client from a custom credentials and a GcpCallInvoker that has
+            // non-default gRPC stream/channel options.
+            // Note: the GcpCallInvoker should be long-lived while the BigtableClient
+            // instances can be emphemeral and each use the same call invoker
```

Comment:
Nit: typo for ephemeral. Possibly add a period too.

Reply:
Fixed